src/lib/services/expression/proposalExpression.ts

```diff
@@ -0,0 +1,24 @@
+import { useWallet } from "@cosmos-kit/react";
+
+import type { ProposalStatus, ProposalType } from "lib/types";
+
+export const useProposalListExpression = (
+  statuses: ProposalStatus[],
+  types: ProposalType[],
+  search: string,
+  isMyProposal: boolean
+) => {
+  const { address } = useWallet();
```

Comment:
receive as props instead so this wont be a hook

Reply:
Agreed, just make it a utility function.